scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,88 @@
+
+# confirm a result of `issue_nft`
+key=$(cat ./tmp.json | jq '.logs[] | .events[0]' | jq '.attributes[0]' | jq '.key')
+value=$(cat ./tmp.json | jq '.logs[] | .events[0]' | jq '.attributes[0]' | jq '.value')
+expected_key="\"_contract_address\""
+expected_val="\"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8\""
+test "${expected_key}" = "${key}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+test "${expected_val}" = "${value}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+
+#*** mint_nft collection contract ***
+# mint_nft
+fnsad tx wasm execute link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8 '{"mint_nft":{"from":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","to":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","params":[{"token_type":"10000001","name":"nft1_name1","meta":"nft1_meta1"},{"token_type":"10000001","name":"nft1_name2","meta":"nft1_meta2"}]}}' --from link146asaycmtydq45kxc8evntqfgepagygelel00h --node http://localhost:26658 --home=${chain_dir} --chain-id simd-testing --keyring-backend test --gas 10000000 -b block -o json -y > tmp.json
+
+# confirm a result of `mint_nft`
+key=$(cat ./tmp.json | jq '.logs[] | .events[0]' | jq '.attributes[0]' | jq '.key')
+value=$(cat ./tmp.json | jq '.logs[] | .events[0]' | jq '.attributes[0]' | jq '.value')
+expected_key="\"_contract_address\""
+expected_val="\"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8\""
+test "${expected_key}" = "${key}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+test "${expected_val}" = "${value}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+
+#*** send_nft collection contract ***
+# send_nft
+fnsad tx wasm execute link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8 '{"send_nft":{"from":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","to":"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8","token_ids":["1000000100000001"]}}' --from link146asaycmtydq45kxc8evntqfgepagygelel00h --keyring-backend test --node http://localhost:26658 --home=${chain_dir} --chain-id simd-testing --gas 10000000 -b block -o json -y > tmp.json
+
+# confirm a result of `send_nft`
+key=$(cat ./tmp.json | jq '.logs[] | .events[0]' | jq '.attributes[0]' | jq '.key')
+value=$(cat ./tmp.json | jq '.logs[] | .events[0]' | jq '.attributes[0]' | jq '.value')
+expected_key="\"_contract_address\""
+expected_val="\"link14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sgf2vn8\""
+test "${expected_key}" = "${key}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+test "${expected_val}" = "${value}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+
+#*** burn_nft collection contract ***
+# burn_nft
```

Comment:
It is the part of the above line. is this needed?

Reply:
Not needed. I deleted it.